src/eeg_series.jl

```diff
@@ -87,10 +90,12 @@ function eeg_topoplot_series!(
     label = :label,
     col = :time,
     row = nothing,
-    combinefun = mean,
     col_labels = false,
     row_labels = false,
-    rasterize_heatmap = true,
+    rasterize_heatmaps = true,
```

Comment:
strictly speaking, this renaming makes it breaking

Reply:
what is actually better: rasterize_heatmap or rasterize_heatmaps?
Because it was rasterize_heatmaps everywhere except this line

Reply:
it is one heatmap in this case, so I guess singular here?

I guess you refer to toposeries, there you have multiple heatmaps